src/main/kotlin/elan/tweaks/thaumcraft/research/frontend/integration/ThaumcraftResearchTweaks.kt

```diff
@@ -4,44 +4,32 @@ import cpw.mods.fml.common.Mod
 import cpw.mods.fml.common.SidedProxy
 import cpw.mods.fml.common.event.FMLLoadCompleteEvent
 import cpw.mods.fml.common.network.NetworkRegistry
-import elan.tweaks.thaumcraft.research.frontend.integration.ThaumcraftResearchTweaks.DEPENDENCIES
 import elan.tweaks.thaumcraft.research.frontend.integration.ThaumcraftResearchTweaks.MOD_ID
-import elan.tweaks.thaumcraft.research.frontend.integration.ThaumcraftResearchTweaks.MOD_LANGUAGE_ADAPTER
-import elan.tweaks.thaumcraft.research.frontend.integration.ThaumcraftResearchTweaks.NAME
-import elan.tweaks.thaumcraft.research.frontend.integration.ThaumcraftResearchTweaks.VERSION
 import elan.tweaks.thaumcraft.research.frontend.integration.proxies.SingletonInitializer
 import elan.tweaks.thaumcraft.research.frontend.integration.table.ThaumcraftResearchGuiHandler
 
 @Mod(
     modid = MOD_ID,
-    name = NAME,
-    version = VERSION,
-    dependencies = DEPENDENCIES,
-    modLanguageAdapter = MOD_LANGUAGE_ADAPTER
-)
+    name = "Thaumcraft Research Tweaks",
+    version = MODVER,
+    modLanguageAdapter = "net.shadowfacts.forgelin.KotlinAdapter",
+    dependencies = "required-after:forgelin;required-after:spongemixins;required-after:Thaumcraft;")
```

Comment:
I am trying to make spotless stop complaining about non constant in annotation. However issue might be in a different thing. MODVER is generated at some point (I am not sure which task does it) and in @Dream-Master PR it fails to generate. Maybe a change in buildscript is required, not sure.